Makefile

```diff
@@ -62,27 +62,26 @@ $(PYTHON_DEPENDENCY_FILE): $(REQUIREMENTS_FILES)
 	touch $(PYTHON_DEPENDENCY_FILE)
 
 
-.PHONY: lint                                                                            ## Lints the code for all available runtimes
-lint: lint-templates lint-sam-apps
+lint: lint-all																			## Lints the code for all available runtimes
+	@echo "CFN Lint complete"
+.PHONY: lint
 
 
-.PHONY: lint-templates
-lint-templates: $(CFN_LINT_OUTPUT)
-$(CFN_LINT_OUTPUT): $(ALL_CFN_TEMPLATES) $(TEMPLATE_FILE)
-	for t in $? ; do \
-		cfn-lint -t $${t} ; \
-	done
+lint-all: $(ALL_CFN_TEMPLATES)
```

Comment:
Why this way? Because the for loops, executed by the shell are hard to read and eat failures and don't cause Make to fail when tests fail

Reply:
very exciting that @Tankanow has tortured you into learning Make. This looks like a good change to me 👍 

I don't work here, but _technically_ this is an open source project :)

Reply:
This is so beautiful!

Reply:
😂

Reply:
Hi @imcgaunn!

Reply:
Hi @Tankanow - I think since I've committed to this project (maybe?) I am subscribed to all updates from it. The sight of @silvexis  committing a  `Makefile` change was just too much for me to ignore